lib/omnitrade_api/client.rb

```diff
@@ -71,8 +72,8 @@ def setup_auth_keys(options)
         @access_key = options[:access_key]
         @secret_key = options[:secret_key]
         @auth       = Auth.new @access_key, @secret_key
-      else
-        #raise ArgumentError, 'Missing access key and/or secret key'
+        # else
+        # raise ArgumentError, 'Missing access key and/or secret key'
```

Comment:
Can you check with the dev who commented this line to maybe delete it in this PR? :thinking:

Reply:
@rhiroyuki this was present in the original Peatio fork, and ended up here as well

I've removed/improved this with https://github.com/OmniTrade/client-ruby/pull/20